### ISSUE TITLE
Downstream finetuning application

### ISSUE THREAD

**Issue by chl916185** [kang-wu/SkySensePlusPlus#4]
Your work is excellent. I looked at the pre-training code, and SwinTransformerV2 was the focus of the training. Why wasn't SwinTransformerV2 chosen for downstream tasks instead of VisionTransformer?
@kang-wu

**Comment by kang-wu** [comment]
Thank you for the excellent question.

The model selection for downstream tasks is based on data modality. We use ViT for multispectral/SAR images and Swin Transformer V2 for high-resolution RGB images.

The current public evaluation code only includes tasks on multispectral and SAR data, which is why the ViT application is demonstrated. The code for downstream tasks using SwinV2 is currently being refactored and will be included in a recent update.

Stay tuned, and thank you for the valuable feedback!

**Comment by yizw0011** [comment]
Does the jl16 dataset released on Huggingface only contain images without annotations?

**Comment by kang-wu** [comment]
> Does the jl16 dataset released on Huggingface only contain images without annotations?

Thank you for your question. Currently, the labels and s1 data are hosted on the Zenodo project and have not yet been migrated to a single repository.
